src/main/java/seedu/address/logic/commands/SetConsultationFeeCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.math.BigDecimal;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.record.Statistics;
+
+/**
+ * Sets the consultation fee of the clinic.
+ */
+public class SetConsultationFeeCommand extends Command {
+
+    public static final String COMMAND_WORD = "setconsultfee";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":sets the consultation fee to the desired value.\n"
+            + "Parameters: FEE\n"
+            + "Example: " + COMMAND_WORD + " 30.00";
+
+    public final BigDecimal fee;
+
+    public SetConsultationFeeCommand(BigDecimal fee) {
+        requireNonNull(fee);
+        this.fee = fee;
+    }
+
+    @Override
+    public CommandResult execute(Model model, CommandHistory history) throws CommandException {
+        requireNonNull(model);
+        model.setConsultationFee(fee);
```

Comment:
Should have a try and catch block to convert exception generated by Model to CommandException

Reply:
Added in the try catch block.